tests/unit/parser.js

```diff
@@ -2716,6 +2740,21 @@ exports["test destructuring function as legacy JS"] = function (test) {
   test.done();
 };
 
+exports["destructuring function default values"] = function (test) {
+  var code = [
+    "function a([ b = 2 ]) {}",
```

Comment:
I am pretty sure we don't, but we do have some nonsense tests about default arguments in core :<